articles/reporting/engagement-recency.md

```diff
@@ -0,0 +1,55 @@
+
+_**Note:** For Engagement Recency to function, you must enable at open or click tracking. For best results, enable both._
+
+## Engagement Recency Cohorts
+
+![Engagement Recency chart](media/engagement-recency/chart.png)
+
+Recipients are divided into the following cohorts:
+* **New Recipients** – Recipients who have received their first email from you within the past 7 days and haven't yet engaged with your messages
+* **Never Engaged** – Recipients who have received email have never engaged with your emails or they last engaged more than a year ago.
+* **Not Recently Engaged** – Recipients who have engaged with your email in the past 365 days, but not in the past 90 days.
+* **Semi-Recently Engaged** – Recipients who have engaged with your email in the past 90 days, but not in the past 14 days.
+* **Recently Engaged** – Recipients who have engaged with your email in the past 14 days.
+
+
+## Dashboard
+
+On the Engagement Recency dashboard you'll find table of the raw counts or ratios of the mail you've sent to recently engaged recipients. You should strive to have this cohort be the largest. Sort by the total column on the right to find your best and worst performing facets. To find specific issues, group the results by subaccount, and any IP pool, sending domain, or campaign. Quickly focus using a preset or custom time range.
+
+![Engagement Recency dashboard](media/engagement-recency/dashboard.png)
+
+## Details page
+
+On each details page you'll find 4 different charts, each in a different tab.
+
+The first chart shows the total emails you sent each day, broken up by each cohort.
+
+In the example below, you can see 51.2% of the email that were sent on August 16th were sent to recipients who engaged with this sender's emails in the past 14 days.
+
+![Engagement Recency details page](media/engagement-recency/details.png)
+
+With this report, you can see if you are sending to too many disengaged recipients. If the Never Engaged and Not Recently Engaged cohorts make up a majority of the people you send to, you should examine your [list hygiene practices](https://www.sparkpost.com/docs/signals/list-hygiene/).
+
+### Behavior by Cohort
+
+Knowing which cohorts your sending to is an important step in understanding the health of your email program. You can further dive into each group of recipients to optimize your sending and messaging, as well as find areas for improvement. The three Behavior by Engagement Cohorts reports provide metrics about how each of your different cohorts are engaging with your email, as well as suggestions on how to interpret them. For example, if you see a high unsubscribe or complaint rate from the New cohort, you probably need to work on your list collection and set better expections with new recipients when you collect their emails.
```

Comment:
- I'd lose the first sentence / Behavior by Engagement Cohorts
 
You can further dive into each group of recipients to optimize your sending and find areas for improvement. The next three reports show how your different cohorts are engaging with your emails, as well as recommendations on how to interpret their behavior. For example, if you see a high unsubscribe or complaint rate from the New cohort, you probably need to work on your list collection and set better expectations with new recipients when you collect their emails.

Reply:
I think also maybe the recommendations could either be omitted or have their own section.